templates/config/nginx/default.conf.mustache

```diff
@@ -0,0 +1,185 @@
+
+		set_escape_uri $escaped_key $key;
+
+		srcache_fetch GET /redis-fetch $key;
+		srcache_store PUT /redis-store key=$escaped_key;
+
+		more_set_headers 'X-SRCache-Fetch-Status $srcache_fetch_status';
+		more_set_headers 'X-SRCache-Store-Status $srcache_store_status';
+
+		include fastcgi_params;
+		fastcgi_pass php:9000;
+	}
+
+	{{/include_redis_conf}}
+
+	{{#include_wpsubdir_conf}}
+	# WPSUBDIRECTORY NGINX CONFIGURATION
+	if (!-e $request_filename) {
+
+		# Redirect wp-admin to wp-admin/
+		rewrite /wp-admin$ $scheme://$host$uri/ permanent;
+
+		# Redirect wp-* files/folders
+		rewrite ^(/[^/]+)?(/wp-.*) $2 last;
+
+		# Redirect other php files
+		rewrite ^(/[^/]+)?(/.*\.php) $2 last;
+	}
+	{{/include_wpsubdir_conf}}
+
+	{{#include_php_conf}}
+	# PHP NGINX CONFIGURATION
+	location / {
+		try_files $uri $uri/ /index.php?$args;
+	}
+	location ~ \.php$ {
+		try_files $uri =404;
+		fastcgi_param  SCRIPT_FILENAME  $document_root$fastcgi_script_name;
+		include fastcgi_params;
+		fastcgi_pass php:9000;
+	}
+	{{/include_php_conf}}
+
+	{{! wpcommon.conf }}
+	# WordPress COMMON SETTINGS
+	# Limit access to avoid brute force attack
+	location = /wp-login.php {
+		limit_req zone=one burst=1 nodelay;
+		include fastcgi_params;
+		fastcgi_pass php:9000;
+	}
+	# Disable wp-config.txt
+	location = /wp-config.txt {
+		deny all;
+		access_log off;
+		log_not_found off;
+	}
+	# Disallow php in upload folder
+	location /wp-content/uploads/ {
+		location ~ \.php$ {
+			#Prevent Direct Access Of PHP Files From Web Browsers
+			deny all;
+		}
+	}
+	{{! /wpcommon.conf }}
+
+	{{! locations.conf }}
+	# NGINX CONFIGURATION FOR COMMON LOCATION
+	# Basic locations files
+	location = /favicon.ico {
+		access_log off;
+		log_not_found off;
+		expires max;
+	}
+
+	location = /robots.txt {
+	# Some WordPress plugin gererate robots.txt file
+	# Refer #340 issue
+	return 200 'User-agent: *\nDisallow: /';
+		access_log off;
+		log_not_found off;
+	}
+	# Cache static files
+	location ~* \.(ogg|ogv|svg|svgz|eot|otf|woff|mp4|ttf|rss|atom|jpg|jpeg|gif|png|ico|zip|tgz|gz|rar|bz2|doc|xls|exe|ppt|tar|mid|midi|wav|bmp|rtf|swf)$ {
+		add_header "Access-Control-Allow-Origin" "*";
+		access_log off;
+		log_not_found off;
+		expires max;
+	}
+	# Security settings for better privacy
+	# Deny hidden files
+	location /.well-known {
+		allow all;
+	}
+	location ~ /\. {
+		deny all;
+		access_log off;
+		log_not_found off;
+	}
+	# Deny backup extensions & log files
+	location ~* ^.+\.(bak|log|old|orig|original|php#|php~|php_bak|save|swo|swp|sql)$ {
+		deny all;
+		access_log off;
+		log_not_found off;
+	}
+	# Return 403 forbidden for readme.(txt|html) or license.(txt|html) or example.(txt|html)
+	if ($uri ~* "^.+(readme|license|example)\.(txt|html)$") {
+		return 403;
+	}
+	# Status pages
+	location /nginx_status {
+		stub_status on;
+		access_log off;
+	}
+	location ~ ^/(status|ping) {
+		include fastcgi_params;
+		fastcgi_pass php:9000;
+	}
+	location ~*  \.(css|js)$ {
```

Comment:
@kirtangajjar remove this `location` block at all. A 5s cache is no useful.

Is there way we add `--env` check? In dev env this block should not exist. In production, this can exist with longer expiry.

Reply:
Dev env will be handled in beta2. Please assume all config is for prod. We'll add flag as well as separate config for dev in next release. @rahul286

Reply:
Ok for production, make 5s atleadt 1d.